data_filter_mongodb/Makefile

```diff
@@ -0,0 +1,38 @@
+VERSION := "0.1"
+GIT_COMMIT := $(shell git rev-parse --short HEAD)
+GO := go
+
+CGO_ENABLED ?= 0
+# Force modules on and to use the vendor directory.
+GO := CGO_ENABLED=$(CGO_ENABLED) GO111MODULE=on GOFLAGS=-mod=vendor go
+
+######################################################
+#
+# Development targets
+#
+######################################################
+
+
+all: build test
+
+LDFLAGS := -X github.com/open-policy-agent/contrib/data_filter_mongodb/internal/version.Vcs=$(GIT_COMMIT) \
+	-X github.com/open-policy-agent/contrib/data_filter_mongodb/internal/version.Version=$(VERSION)
+
+build:
+	go build -o bin/opa-mongo github.com/open-policy-agent/contrib/data_filter_mongodb
+
+vet:
+	go vet ./...
+
+fmt:
+	go fmt ./...
+
+lint:
+	golangci-lint run
+
+test:
+	$(GO) test ./...
+
+all: build vet fmt lint test
+
```

Comment:
We should add a `push` target that builds and pushes the image. This target will be called from the top-level. See [this](https://github.com/open-policy-agent/contrib/blob/master/envoy_iptables/Makefile) for an example.